### DISCUSSION TITLE
Weird issue with Compiler

### DISCUSSION THREAD

**Discussion by AlexEdimensionz** [joomengine/Joomla-Component-Builder#944] [Extention Issues] [answered]
I have even tried recreating my component from scratch with a fresh install of Joomla and JCB
The compiler "completes" but the zip is incomplete. Missing XML, many times nothing but the **site** and **admin** folders.

Has anyone else run into this issue?

It happens with Joomla 3x - both PHP8.1 and 7.4  (JCB v3.1.12)

The compile only completes if i add a new field to an Admin view, but only about 50% of the time. Recompiling an existing project fails.

**Comment by Llewellynvdm** [comment, marked as answer]
[Follow issue here!](https://git.vdm.dev/joomla/Component-Builder/issues/968)

**Comment by jsr4id** [comment]
This is still hapening on Joomla 5 and JCB 5.0.0.
Is there any solution for this? I already read the linked article.

**Reply by jsr4id** [reply]
For anyone who has a similar problem. The developer recommends using the Firefox browser. Chrome and Edge seem to cause problems with the compiler.

Using Firefox helped me.
